vocab_files/custom_vocabs/basal-sweep-hit-types.ttl

```diff
@@ -35,6 +22,18 @@ PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
     tern:valueType tern:IRI ;
 .
 
+<https://linked.data.gov.au/def/test/dawe-cv/84b0a152-e6f6-4ea2-b973-04238034bb52>
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/custom_vocabs/basal-sweep-hit-types.ttl" ;
```

Comment:
Why is this different to the code you deleted in line 12-14?

Reply:
If it's already in `vocab_files/observable_properties_by_module/basal-area/basal-wedge-protocol/collection.ttl` then we don't need this here anymore.